app/code/Magento/Contact/Controller/Index/Post.php

```diff
@@ -101,4 +65,61 @@ private function getDataPersistor()
 
         return $this->dataPersistor;
     }
+
+    /**
+     * @param array $post Post data from contact form
+     * @return void
+     */
+    protected function sendEmail($post)
+    {
+        $storeScope = \Magento\Store\Model\ScopeInterface::SCOPE_STORE;
+        $transport = $this->_transportBuilder
+            ->setTemplateIdentifier($this->scopeConfig->getValue(self::XML_PATH_EMAIL_TEMPLATE, $storeScope))
+            ->setTemplateOptions(
+                [
+                    'area' => \Magento\Backend\App\Area\FrontNameResolver::AREA_CODE,
+                    'store' => \Magento\Store\Model\Store::DEFAULT_STORE_ID,
```

Comment:
yeah, noticed that, though the fact you touched that classed triggered static test failure :) I'd appreciate if you ensure that the dependency is needed and add it to the composer.